packages/core/src/types/index.ts

```diff
@@ -82,6 +82,12 @@ import type { PixelDataTypedArray } from './PixelDataTypedArray';
 import type { ImagePixelModule } from './ImagePixelModule';
 import type { ImagePlaneModule } from './ImagePlaneModule';
 import type { AffineMatrix } from './AffineMatrix';
+export type {
+  RetrieveStage,
+  RetrieveOptions,
+  IRetrieveConfiguration,
+} from './IRetrieveConfiguration';
```

Comment:
why we have a mix of type and interface here? can't we simply have `RetrieveConfiguration` (as type) too?